app/templates/components/orders/order-summary.hbs

```diff
@@ -110,7 +110,11 @@
                         {{t 'The price is inclusive of all taxes.'}}
                       {{else}}
                         {{#if ticket.price}}
-                          {{this.event.tax.rate}}%
+                        <td><CurrencyAmount @currency={{this.eventCurrency}} @amount={{sub ticket.ticketPriceWithTax ticket.price}}/>
+                 <small class="ui gray-text small">
+                          ({{this.event.tax.name}} {{this.event.tax.rate}}%)
+                        </small>
```

Comment:
6th time I guess - indentation.  [how to format code](https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode)

Reply:
sorry it is do code formatter in my sublime.... and it is previous commit.